src/index.sh

```diff
@@ -4,7 +4,7 @@ set -e
 # shellcheck source=src/git_exists.sh
 source "$(dirname "${BASH_SOURCE[0]}")/git_exists.sh"
 
-if ! git_exists "https://$INPUT_TOKEN@$INPUT_REPOSITORY.wiki.git" &>/dev/null; then
+if ! git_exists "https://x:$INPUT_TOKEN@$INPUT_REPOSITORY.wiki.git" &>/dev/null; then
```

Comment:
Even if this value is useless, I'd like it to default to the current user or something. However, before addressing this, check my other comment.